src/app/core/components/event-tag/event-tag.component.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../../../styles-variables-themes.scss';
+@import '../../../../styles-variables.scss';
+
+.tag-card {
+  display: flex;
+  padding: 5px 7px !important;
+  margin: 5px;
+  max-height: 24px;
+  border-radius: 2px !important;
+  background: $yellow-light !important;
```

Comment:
create event-tag.component-theme.scss (like the one in pagination component)
and it will be easier to override style without !important